module/documents/chat-message.mjs

```diff
@@ -331,19 +353,27 @@ export default class ChatMessage5e extends ChatMessage {
     const attackRoll = this.rolls[0];
     const targets = this.getFlag("dnd5e", "targets");
     if ( !game.user.isGM || !(attackRoll instanceof dnd5e.dice.D20Roll) || !targets?.length ) return;
-    const evaluation = document.createElement("ul");
-    evaluation.classList.add("dnd5e2", "evaluation");
-    evaluation.innerHTML = targets.map(({ name, img, ac, uuid }) => {
+    const tray = document.createElement("div");
+    tray.classList.add("dnd5e2");
+    tray.innerHTML = `
+      <div class="card-tray targets-tray collapsible collapsed">
```

Comment:
With the auto-expanding chat trays, this results in the tray being expanded for a single target in addition to the highlighting of the total result.

Reply:
Do you think we should make a special exception for the attack targets tray? With the new condensed display this is probably not a problem anymore. It perhaps obviates the need to even show success/failure on the attack roll anymore but it feels reasonable to me to have consistency across attack rolls, ability checks and saving throws now with an appropriate target. What do you think?